Package.swift

```diff
@@ -43,6 +44,7 @@ let package = Package(
                 .product(name: "SourceKittenFramework", package: "SourceKitten"),
                 "SwiftSyntax",
                 "Yams",
+                "Pathos"
```

Comment:
Nit: this list is sorted, can you move this to the top of the dependencies?

Reply:
Fixed: [85841c2](https://github.com/realm/SwiftLint/pull/3843/commits/85841c2e391755766d17728da2aed6810244d08d)